lib/Config/ServicesChecker.php

```diff
@@ -122,7 +122,8 @@ public function isPremiumKeyValid($displayErrorNotice = true) {
   public function isMailPoetAPIKeyPendingApproval(): bool {
     $mssActive = Bridge::isMPSendingServiceEnabled();
     $mssKeyValid = $this->isMailPoetAPIKeyValid();
-    $mssKeyPendingApproval = $this->settings->get('mta.mailpoet_api_key_state.data.is_approved') === false;
+    $isApproved = $this->settings->get('mta.mailpoet_api_key_state.data.is_approved');
```

Comment:
I guess this fix has to be in this place since some users already might have it saved like that, correct?

Reply:
Yes, but also - the problem still exists. It will be fixed only by converting the API to JSON.